### DISCUSSION TITLE
Are merged snow fall amounts under gridpoints cumulative?

### DISCUSSION THREAD

**Discussion by wsamoht** [weather-gov/api#266] [Q&A]
I am wondering if the `snowfallAmount` values under the `gridpoints` endpoint are cumulative or not when the consecutive values are merged. Looking at the data it would seem that they are but wanted to confirm. I would also assume `iceAccumulation` and `quantitativePrecipitation` are also cummulative?

Here is example data I am looking at from 10/21/2020. The endpoint used was `https://api.weather.gov/gridpoints/DLH/62,92`. I removed `0` values to keep it concise.

```json
{
  "snowfallAmount": {
    "uom": "wmoUnit:mm",
    "values": [
      {
        "validTime": "2020-10-22T12:00:00+00:00/PT6H",
        "value": 63.5
      },
      {
        "validTime": "2020-10-22T18:00:00+00:00/PT6H",
        "value": 116.84
      },
      {
        "validTime": "2020-10-23T00:00:00+00:00/PT6H",
        "value": 27.94
      }
    ]
  }
}
```

Adding the values up assuming they are cumulative equals 208.28mm of snow fall or 8.2in which is consistent with the Winter Weather Advisory (see below). If they are not cumulative, we would have 1249.68mm or 49.2in.

> ...NEXT ROUND OF ACCUMULATING SNOW EXPECTED TODAY...
> 
> .Our next round of accumulating snow will arrive today, bringing
several inches of snow to portions of the Northland. A strong
area of low pressure has developed over Colorado, leading to the
increased chances of accumulating snow. The forecast generally
remains on track since yesterday afternoon, with the potential
for a swath of accumulations between **5 to 8 inches** from the
Brainerd Lakes region towards the Twin Ports. Some uncertainty
remains on the southern edge of the forecast snowfall swath as
there will be a transition to a wintry mix or all rain the further
southeast you go, so there is the potential for a shift in the
location of the heavier snowfall. Some sleet or freezing rain may
mix in with the snow at times at this transition zone over
portions of northwest Wisconsin.
> ...

**Comment by JJKraw** [comment]
Not an authoritative answer, but from my observations, the amount pertains to how much is expected to fall only during the time period indicated by validTime. So, as you also saw, adding them up should give the total snowfall expected. Unless for some reason two time periods overlapped (which I've never seen).

**Comment by netbymatt** [comment]
I watched these numbers fairly closely last winter and what I learned is to read it like this:
63mm of snow will fall between 12:00 and 18:00.
116mm of snow will fall between 18:00 and 00:00... etc.
This seemed to match fairly well with what other forecast products and watches also said. The ice and precipitation values seemed to follow that pattern too.
I like being able to read it this way because it also gives you an idea of snow/ice/rainfall intensity.

**Comment by wsamoht** [comment]
Awesome! Thank you @JJKraw and @netbymatt. Great to have confirmation from other users. I will consider the three of us seeing it being cumulative be a confirmation it is indeed. That is the way I programmed it and it is already in production :)
